From 7784d2ad28219b4cc0007261b36b6c85d1180c2f Mon Sep 17 00:00:00 2001
From: Jianzheng Zhou <jianzheng.zhou@freescale.com>
Date: Tue, 23 Jul 2013 15:55:15 +0800
Subject: [PATCH 218/281] ENGR00270791 build:fix build error

update realtek wifi device name to UNITE to allow work together with
atheros in one image.

Change-Id: Ib19b2459a63541c9771e7398351af9937c36bef9
Signed-off-by: Jianzheng Zhou <jianzheng.zhou@freescale.com>
---
 hdmidongle_6dq/BoardConfig.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hdmidongle_6dq/BoardConfig.mk b/hdmidongle_6dq/BoardConfig.mk
index 4467489..c3736dd 100755
--- a/hdmidongle_6dq/BoardConfig.mk
+++ b/hdmidongle_6dq/BoardConfig.mk
@@ -23,7 +23,7 @@ BOARD_WPA_SUPPLICANT_DRIVER := NL80211
 BOARD_WPA_SUPPLICANT_PRIVATE_LIB := lib_driver_cmd_rtl
 BOARD_HOSTAPD_DRIVER := NL80211
 BOARD_HOSTAPD_PRIVATE_LIB := lib_driver_cmd_rtl
-BOARD_WLAN_DEVICE := rtl8192cu
+BOARD_WLAN_DEVICE := UNITE
 #BOARD_WLAN_DEVICE := rtl8192du
 #BOARD_WLAN_DEVICE := rtl8192ce
 #BOARD_WLAN_DEVICE := rtl8192de
-- 
1.8.0

